tiltak/src/test/resources/deltaker-statistikk-repository_test-data.sql

```diff
@@ -0,0 +1,73 @@
+DO $$
+DECLARE
+    brukerId1 uuid := '23b04c3a-a36c-451f-b9cf-30b6a6b586b8';
+    deltaker1 uuid := 'dc600c70-124f-4fe7-a687-b58439beb214';
+
+    brukerId2 uuid := '82536415-da79-4654-91c6-b68ddf68da16';
+    deltaker2 uuid := '25b3ef51-02ff-4f26-8f1b-1ad9215458d3';
+
+    brukerId3 uuid := 'cd8930fe-b29b-4a7d-ae90-d1fd9e462f06';
+    deltaker3 uuid := '8b822579-a7af-4f6a-91af-416c08e29a62';
+
+    arrangor1 uuid := '8a37bce6-3bc1-11ec-8d3d-0242ac130003';
+    arrangor2 uuid := 'e5f0725d-d0f9-4660-8e53-64e00ceeb7c9';
+    ansatt1 uuid := '90f5bda4-28b6-41ba-9385-86d5e0e27103';
+
+BEGIN
```

Comment:
Det har blitt merget inn noen nye endringer som gjør det mye enklere å inserte data for tester, da slipper man heavy sql scripts